README.rst

```diff
@@ -92,6 +92,39 @@ The library is tested with these specific, common types:
 
 * ``code39`` - `Code 39`_
 
+Command-line interface
+----------------------
+
+Treepoem also includes a simple command-line interface to the
+functionality of ``generate_barcode``. For example, these commands
+will generate two QR codes with identical contents, but different levels
+of error correction (see `QR Code Options`_):
+
+.. code-block:: sh
+
+   $ treepoem -o barcode.png -t qrcode "This is a test" eclevel=H
+   $ treepoem -o barcode.png -t qrcode "This is a test" eclevel=L
+
+Complete usage is shown with ``treepoem --help``:
+
+.. code-block:: sh
```

Comment:
I don't think we need to copy/paste the output into the README, that just creates maintenance burden. Just leaving a mention to check `treepoem --help` is enough.